.github/workflows/functional_all_db.yml

```diff
@@ -44,9 +44,6 @@ jobs:
     - uses: actions/setup-dotnet@v3
       with:
         dotnet-version: '7.0.x'
-    - name: Fix nuget restore issue
```

Comment:
This was a old bug in NuGet service related to SSL certificates. This is not a problem aymore.